src/lib/functions/form-submissions-handler.mjs

```diff
@@ -29,7 +29,12 @@ const getFormHandler = function ({ functionsRegistry }) {
 
 export const createFormSubmissionHandler = function ({ functionsRegistry, siteUrl }) {
   return async function formSubmissionHandler(req, res, next) {
-    if (req.url.startsWith('/.netlify/') ||req.method !== 'POST' || (await functionsRegistry.getFunctionForURLPath(req.url, req.method))) return next()
+    if (
+      req.url.startsWith('/.netlify/') ||
+      req.method !== 'POST' ||
+      (await functionsRegistry.getFunctionForURLPath(req.url, req.method))
```

Comment:
Is it possible that `functionsRegistry` isn't defined here? It's a bit hard to tell, but we do have a few checks for it elsewhere in the codebase, which makes me think it may not be there at times.

Reply:
From what i'm seeing, this is the only codepath where this is ever going to be called:

https://github.com/netlify/cli/blob/62e0ef553eb17c872099f25b3e139eecf04aba1d/src/lib/functions/server.mjs#L276-L290

So `functionsRegistry` is always defined.

Reply:
👍🏻 🚀